helix-common/src/main/java/org/apache/helix/HelixException.java

```diff
@@ -30,6 +30,16 @@ public HelixException(String message) {
     super(message);
   }
 
+  /**
+   * Create a HelixException that can optionally turn off stack trace. Its other characteristics are
+   * the same as a HelixException with a message.
+   * @param message the detail message
+   * @param writableStackTrace whether or not the stack trace should be writable
+   */
+  public HelixException(String message, boolean writableStackTrace) {
```

Comment:
So RuntimeException can be configured not to have stack trace out.

Reply:
Yes sir